src/components/organisms/FileTreePane/TreeItem.tsx

```diff
@@ -0,0 +1,282 @@
+            disabled={isInPreviewMode || (!isFolder && !isSupported && !isExcluded)}
+            onClick={e => {
+              e.domEvent.stopPropagation();
+              if (isExcluded) {
+                onIncludeToProcessing(relativePath);
+              } else {
+                onExcludeFromProcessing(relativePath);
+              }
+            }}
+            key="add_to_files_exclude"
+          >
+            {isExcluded ? 'Remove from' : 'Add to'} Files: Exclude
+          </Menu.Item>
+        </>
+      ) : null}
+      <S.ContextMenuDivider />
+      <Menu.Item
+        onClick={e => {
+          e.domEvent.stopPropagation();
+          navigator.clipboard.writeText(absolutePath);
+        }}
+        key="copy_full_path"
+      >
+        Copy Path
+      </Menu.Item>
+      <Menu.Item
+        onClick={e => {
+          e.domEvent.stopPropagation();
+
+          navigator.clipboard.writeText(relativePath);
+        }}
+        key="copy_relative_path"
+      >
+        Copy Relative Path
+      </Menu.Item>
+      {fileMap[ROOT_FILE_ENTRY].filePath !== treeKey ? (
+        <>
+          <S.ContextMenuDivider />
+          <Menu.Item
+            disabled={isInPreviewMode}
+            onClick={e => {
+              e.domEvent.stopPropagation();
+              onRename(absolutePath, osPlatform);
+            }}
+            key="rename_entity"
+          >
+            Rename
+          </Menu.Item>
+          <Menu.Item
+            disabled={isInPreviewMode}
+            key="delete_entity"
+            onClick={e => {
+              e.domEvent.stopPropagation();
+              deleteEntityWizard(
+                {entityAbsolutePath: absolutePath},
+                () => {
+                  setProcessingEntity({processingEntityID: treeKey, processingType: 'delete'});
+                  deleteEntity(absolutePath, onDelete);
+                },
+                () => {}
+              );
+            }}
+          >
+            Delete
+          </Menu.Item>
+        </>
+      ) : null}
+      <S.ContextMenuDivider />
+      <Menu.Item
+        onClick={e => {
+          e.domEvent.stopPropagation();
+          showItemInFolder(absolutePath);
+        }}
+        key="reveal_in_finder"
+      >
+        Reveal in {platformFilemanagerName}
+      </Menu.Item>
+    </Menu>
+  );
+
+  return (
+    <S.TreeTitleWrapper onMouseEnter={handleOnMouseEnter} onMouseLeave={handleOnMouseLeave}>
+      <S.TreeTitleText>{title}</S.TreeTitleText>
+
+      {processingEntity.processingEntityID === treeKey && processingEntity.processingType === 'delete' ? (
+        <S.SpinnerWrapper>
+          <Spinner />
+        </S.SpinnerWrapper>
+      ) : null}
+      {isTitleHovered && !processingEntity.processingType ? (
+        <>
+          {canPreview(relativePath) && (
+            <S.PreviewSpan
+              type="text"
+              size="small"
+              disabled={isInPreviewMode}
+              isItemSelected={isFileSelected}
```

Comment:
We should put `$` in front of isItemSelected prop used for styling in order to avoid React warnings.

Reply:
Done!